src/js/views/Project/Dependencies.jsx

```diff
@@ -57,11 +60,41 @@ function Dependencies({ project, urlPath }) {
     )
   }
 
+  function getDependency(id) {
+    let name
+    for (const row of rows) {
+      if (row.id === id) {
+        name = row.name
+        break
+      }
+    }
+    return name
+  }
```

Comment:
Does something like `rows.find((row) => row.id == id)?.name` work?

Reply:
Yep that works